test/integration/targets/supervisorctl/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+# output_dir is hardcoded in test/runner/lib/executor.py and created there
+# expand remote path
+- command: 'echo {{ output_dir }}'
+  register: echo
+- set_fact:
+    remote_dir: '{{ echo.stdout }}'
+
+- include_vars: '{{ item }}'
+  with_first_found:
+    - files:
+        - '{{ ansible_distribution }}.yml'
+        - '{{ ansible_os_family }}.yml'
+        - 'defaults.yml'
+
+- block:
```

Comment:
Shouldn't the tasks above be inside this conditional block?

Reply:
updated